app/presenters/licences/set-up.presenter.js

```diff
@@ -28,29 +28,38 @@ const agreementDescriptions = {
  *
  * @param {module:ChargeVersionModel[]} chargeVersions - All charge versions records for the licence
  * @param {module:WorkflowModel[]} workflows - All in-progress workflow records for the licence
- * @param {module:LicenceAgreements[]} agreements - All agreements records for the licence
+ * @param {module:LicenceAgreementModel[]} agreements - All agreements records for the licence
  * @param {module:ReturnVersionModel[]} returnVersions - All returns version records for the licence
  * @param {Object} auth - The auth object taken from `request.auth` containing user details
  * @param {Object} commonData - Licence data already formatted for the view's shared elements
+ * @param {boolean} enableRequirementsForReturns - feature toggle for the return versions links
  *
  * @returns {Object} The data formatted for the view template
  */
-function go (chargeVersions, workflows, agreements, returnVersions, auth, commonData) {
+function go (chargeVersions, workflows, agreements, returnVersions, auth, commonData, enableRequirementsForReturns) {
```

Comment:
We're currently grabbing the feature toggle in `view-licence-set-up.service.js` only to pass it through to the presenter as a param.

We can just grab the feature toggle here in the presenter and not have to change the args.

Reply:
D'oh!